_plugins/jekyll_get.rb

```diff
@@ -15,6 +15,9 @@ def generate(site)
       if !config.kind_of?(Array)
         config = [config]
       end
+      if site.config['host'] == "127.0.0.1"
```

Comment:
Yeah, I'm not sure. I don't think ngrok would affect it unless you overrode it in `_config.yml` or with `--host` on the command line. Now that I think it through, I need to check to see if `host` gets set on `jekyll build` or this would probably stop it on staging/production as well.